src/System.IO.Compression/src/System/IO/Compression/DeflateZLib/Inflater.cs

```diff
@@ -21,19 +21,22 @@ internal sealed class Inflater : IDisposable
         private int _windowBits;                            // The WindowBits parameter passed to Inflater construction
         private ZLibNative.ZLibStreamHandle _zlibStream;    // The handle to the primary underlying zlib stream
         private GCHandle _inputBufferHandle;                // The handle to the buffer that provides input to _zlibStream
+        private long _uncompressedSize;
+        private long _currentInflatedCount = 0;
```

Comment:
0 is already the default field initialization, why specify it explicitly? (in particular: why is this field different than the rest?)

Reply:
I just used to initialize, will remove

> (in particular: why is this field different than the rest?)

Not sure what difference you mean exactly, if about why it is only initilazed then other fields are initialized in the constructors